src/components/onboarding/hooks/useOnBoardingStepsOrder.ts

```diff
@@ -1,6 +1,10 @@
 import { useMemo } from 'react'
 import { useAuth } from 'src/components/auth/AuthContext'
 import { useMyAddress } from 'src/components/auth/MyAccountsContext'
+import {
+  getOffchainToken,
+  isCurrentOffchainAddress,
```

Comment:
it is unclear for me, what is offchain address?

Reply:
Offchain address is an address already integrated with the offchain signer (confirmationless transaction). 

In front-end, users can have several offchain addresses and normal addresses (signing tx). `isCurrentOffchainAddress` is basically checking if current address is offchain address or not.

Reply:
Ask beef for helping with the name, because it hard for understand if you dont follow offchain signer context :)

Reply:
Hi @Yung-Beef 
Please help with suggesting proper names. The description is on my comment above.

Reply:
Yean still not sure I understand @ical10 

Will take a stab at it anyways:
isCurrentProxyAddress
isCurrentOffchainSignerAddress

Are those better @olehmell ?

Reply:
Yeah, sorry for the confusion.

What I'm trying to say with `OffchainAddress` is a normal user address (Talisman, Subwallet, or .js address) that has been connected to offchain signer backend, which happens after two things: acquiring an access token, and adding the main proxy to this address.

For example, `isCurrentOffchainAddress` is just a boolean showing whether an address already "owns" an access token and has added the main proxy. If not, then the account will do tx with confirmation, and vice versa.

Hope this clear things up. @olehmell @Yung-Beef

Reply:
Okay still not entirely sure, but maybe something like

isAddressReadyForOffchain?
isAddressOffchainReady?

@ical10 
@olehmell

Reply:
`isAddressOffchainReady` is good for me.

Reply:
will change to `isAddressOffchainReady` in the next PR.